BraveWallet/WalletStrings.swift

```diff
@@ -1296,5 +1296,103 @@ extension Strings {
+      "wallet.customTokenTitle",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "Custom",
+      comment: "The title displayed on the add custom token screen"
+    )
+    public static let tokenName = NSLocalizedString(
+      "wallet.tokenName",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "Token name",
+      comment: "A title that will be displayed on top of the text field for users to input the custom token name"
+    )
+    public static let enterTokenName = NSLocalizedString(
+      "wallet.enterTokenName",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "Enter token name",
+      comment: "A placeholder for the text field that users will input the custom token name"
+    )
+    public static let tokenContractAddress = NSLocalizedString(
+      "wallet.tokenContractAddress",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "Token contract address",
+      comment: "A title that will be displayed on top of the text field for users to input the custom token contract address"
+    )
+    public static let enterContractAddress = NSLocalizedString(
+      "wallet.enterContractAddress",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "Enter contract address",
+      comment: "A placeholder for the text field that users will input the custom token contract address"
+    )
+    public static let tokenSymbol = NSLocalizedString(
+      "wallet.tokenSymbol",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "Token symbol",
+      comment: "A title that will be displayed on top of the text field for users to input the custom token symbol"
+    )
+    public static let enterTokenSymbol = NSLocalizedString(
+      "wallet.enterTokenSymbol",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "Enter token symbol",
+      comment: "A placeholder for the text field that users will input the custom token symbol"
+    )
+    public static let decimalsPrecision = NSLocalizedString(
+      "wallet.decimalsPrecision",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "Decimals of precision",
+      comment: "A title that will be displayed on top of the text field for users to input the custom token's decimals of precision"
+    )
+    public static let decimalsPrecisionPlaceholder = NSLocalizedString(
+      "wallet.decimalsPrecisionPlaceholder",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "0",
+      comment: "A placeholder for the text field that users will input the custom token's decimals of precision"
+    )
+    public static let customTokenAdd = NSLocalizedString(
+      "wallet.customTokenAdd",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "Add",
+      comment: "The title of the navigation bar item button on the top right of the add custom token screen"
+    )
```

Comment:
We already have this string in WalletStrings (`addAccountAddButton`), maybe just rename that to `add` instead